documentation/pixelclassification/pixelclassification.md

```diff
@@ -35,56 +38,58 @@ Nice properties of the algorithm and workflow are
 
 ## Selecting good features
 
-Assuming the user has already created or loaded an existing
-ilastik project and added a dataset, the first step is to switch to the **Feature Selection Applet**
-where the filter selection and computation are performed.
-The selected features and scales will be used later on for the training of a classifier.
-The selected features and scales should roughly correspond to the visual attributes that separate the
-objects and the background.
+As usual, start by loading the data as described [in the basics]({{site.baseurl}}/documentation/basics/dataselection.html). 
+After the data is loaded, switch to the next applet **Feature Selection**.
+Here you will select the pixel features and their scales which in the next step will be used to discriminate between the different classes of pixels. 
+
 A click on the **Select features** button brings up a feature selection dialog.
 
-<a href="snapshots/feature_selection_zoomed.png" data-toggle="lightbox"><img src="snapshots/feature_selection_zoomed.png" class="img-responsive" /></a>
-<a href="snapshots/feature_selection2.png" data-toggle="lightbox"><img src="snapshots/feature_selection2.png" class="img-responsive" /></a>
+<a href="snapshots/feature_selection_add.png" data-toggle="lightbox"><img src="snapshots/feature_selection_add.png" class="img-responsive" /></a>
+
+For 3D data the features can be computed either in 2D or 3D. 2D can be useful if data has thick slices and the information from a slice is not so relevant for the neighbors. It is also the only way to compute large-scale filters in think stacks. The following image shows the switch between 2D and 3D computation in the Feature Selection dialog.
```

Comment:
I'm guessing it should be "thin stacks" ?

Reply:
Yes, thanks :)

Reply:
```suggestion
For 3D data the features can be computed either in 2D or 3D. 2D can be useful if data has thick slices and the information from a slice is not so relevant for the neighbors. It is also the only way to compute large-scale filters in thin stacks. The following image shows the switch between 2D and 3D computation in the Feature Selection dialog.
```